WriteVHDLSyntax.py

```diff
@@ -1480,6 +1494,7 @@ def writeProcCombination(module, str_ctrl_func, str_ports):
     if "PC_" in module.inst:
         module_str += "  " + module.inst + "_mem_reader : entity work.mem_reader\n"
         module_str += "    generic map (\n"
+        module_str += "      PAGE_LENGTH    => 128,\n"
```

Comment:
@aryd Same comment as above. No need to have a magic 128 here, since the default page length for mem_reader is already 128:
```suggestion
```

Reply:
Removed.